config.yaml

```diff
@@ -5,29 +5,29 @@ native:
   native_grpc: "127.0.0.1:9090"       
 
 ika:
-  ika_rpc: "127.0.0.1:0001"          
-  ika_signer_mnemonic: "" 
-  ika_native_lc_package: ""                
-  ika_native_lc_module: ""               
-  ika_native_lc_function: ""  
-  ika_gas_acc: ""                            
-  ika_gas_budget: ""                         
+  rpc: "http://127.0.0.1:9000"          
```

Comment:
It would be good if we created the demo.config.yaml rather than use "product" config.yaml.

Reply:
Good idea, added a new file with the demo config